### PR TITLE
fix(FBOX): update domain

### LINKED ISSUE #8558: Fboxz.to | fboxz.to - nothing displays, domain changed
### Steps to Reproduce

1. click link (https://fboxz.to/)
2. click movie
3. nothing happens

### Expected Behavior

show movie in presence

### Actual Behavior

nothing happens

### Page URL

https://fboxz.to/

### Screenshots

um i dont think i can lol

### Operating System

W11

### Browser and Version

Chrome Latest (126)

### Additional Comments

The issue is just the fact the url changed from fbox.to to fboxz.to  . thats it lol

### PR DESCRIPTION
**Pull request by itsmeeudrino**
## Description 

This PR updates the FBOX presence's domain from `fbox.to` to `fboxz.to`.
Resolves #8558

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image

</details>

### CODE REVIEW

**Review by DarkVillager — APPROVED** [review]
LGTM

File: websites/F/fbox.to/metadata.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
 	"description": {
 		"en": "fbox online is free movies streaming website with zero ads, it allows you watch free movies online, watch free tv series online in high quality for free"
 	},
-	"url": "fbox.to",
+	"url": "fboxz.to",
```
Comment (DarkVillager) [inline websites/F/fbox.to/metadata.json:21]: no point bcuz of the regex?

**Review by DarkVillager — CHANGES_REQUESTED** [review]
Or they changed how it checks urls/regex urls

File: websites/F/fbox.to/metadata.json (hunk 1 above)
Comment (theusaf) [reply websites/F/fbox.to/metadata.json:21]: Regex doesn't take into account the trailing `z` in the domain

File: websites/F/fbox.to/metadata.json [hunk 2]
```diff
@@ -18,9 +18,9 @@
...
 	"regExp": "(www([0-9]*)?[.])?fbox[.]to",
```
Comment (theusaf) [inline websites/F/fbox.to/metadata.json:22]:
```suggestion
	"regExp": "(www([0-9]*)?[.])?fboxz?[.]to",
```

Comment (Bas950) [reply websites/F/fbox.to/metadata.json:22]: Seems like this would be the best.

**Comment by itsmeeudrino** [comment]
I'm not sure why there's a `regExp` for this presence, would prefer to use the static `url` option if possible

**Comment by DarkVillager** [comment]
> I'm not sure why there's a `regExp` for this presence, would prefer to use the static `url` option if possible

Cuz it changes a lot and there's more than one website?

File: websites/F/fbox.to/metadata.json [hunk 3]
```diff
@@ -18,9 +18,9 @@
 	"description": {
 		"en": "fbox online is free movies streaming website with zero ads, it allows you watch free movies online, watch free tv series online in high quality for free"
 	},
-	"url": "fbox.to",
-	"regExp": "(www([0-9]*)?[.])?fbox[.]to",
-	"version": "1.3.0",
+	"url": "fboxz.to",
+	"regExp": "(www([0-9]*)?[.])?fboxz?[.]to",
```
Comment (DarkVillager) [inline websites/F/fbox.to/metadata.json:22]:
```suggestion
	"regExp": "(www([0-9]*)?[.])?fbox(z)?[.]to",
```

**Comment by itsmeeudrino** [comment]
> Cuz it changes a lot and there's more than one website?

I see, honestly i think a static domain change should require a PR, matching on a domain regex for such site feels like the wrong step to take, perhaps we should make it easier to spot such domain change.

however i don't think this will cause any problems for now

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**